src/plugins/timelist/Timelist.vue

```diff
@@ -57,7 +57,7 @@ const headerItems = [
             if (clock && clock.formatTime) {
                 return clock.formatTime(value);
             } else {
-                return `${moment(value).format(TIME_FORMAT)}Z`;
+                return `${moment.utc(value).format(TIME_FORMAT)}Z`;
```

Comment:
@shefalijoshi - as we discussed I've updated the start / end formatters for the timelist to parse and set to UTC to match the time conductors formatting. This corrects the current activity values which were properly flagged as current but the displayed time value didn't seem to match the current time conductor values.

Reply:
@akhenry + @charlesh88: This design change needs approval to ensure that it meets the intended goal of the timelist. Effectively all start / end time now are displayed as UTC. Previously the timezone offset was included in the timelist start/end values.

Reply:
@michaelrogers please make sure that this change does not break the fix for this issue: https://github.com/nasa/openmct/pull/6495#issuecomment-1481830330